lib/space.rb

```diff
@@ -22,6 +24,11 @@ def self.add(name:, description:, price:, date:)
     Space.create(name: name, description: description, price: price, date: date)
   end
 
+  def book(name)
+    entry = Space.first(:name => name)
```

Comment:
is name unique?

Reply:
It is possible that name could be duplicated in the table, if two listed properties have the same name. There are a couple of ways we could fix this.

Reply:
task added to backlog to use id